app/src/main/java/org/schabi/newpipe/error/ErrorInfo.kt

```diff
@@ -82,10 +82,8 @@ class ErrorInfo(
 
         fun throwableListToStringList(throwableList: List<Throwable>) =
             throwableList.map { it.stackTraceToString() }.toTypedArray()
-
         private fun getInfoServiceName(info: Info?) =
             if (info == null) SERVICE_NONE else ServiceHelper.getNameOfServiceById(info.serviceId)
-
```

Comment:
Remove these unneeded changes.

Reply:
I didn't change this line of code, it was already there, are you sure I want to remove it?

Reply:
In the case you didn't understand, I meant don't remove these lines with your changes in this PR.

Reply:
Oh! make sense!

Reply:
I have modified!